### ISSUE TITLE
How can I train the SfSNet to achieve the paper’s result?

### ISSUE THREAD

**Issue by dldl3D** [senguptaumd/SfSNet#1]
Hi,

I have used the 180k CelebA dataset as the real data(the normal, albedo and light are produced by the pretrained model) and the 180k sysnthetic data the author released. I have trained the model at 35000 iterations, but the training loss of  albedo, light, normal and reconstruction are 320.308, 0.01084, 387.409 and 280.136. But the testing loss of  albedo, light, normal and reconstruction are 715.851, 0.011, 3325.68 and 1513.35. The albedo, normal and reconstruction image are very bad. where is wrong？

The solver file is as follows:
`test_iter: 1000

iter_size: 6

test_interval: 1000

base_lr: 0.01
lr_policy: "multistep"
gamma: 0.1
stepvalue: 15000
stepvalue: 40000
stepvalue: 45000
display: 100
max_iter: 50000
momentum: 0.9
weight_decay: 0.0005
`

**Comment by wujiachen987** [comment]
@dldl3D Are you solve the problem?
